src/sql/azdata.proposed.d.ts

```diff
@@ -1773,6 +1775,35 @@ declare module 'azdata' {
 		}
 	}
 
+	export namespace metadata {
+		export interface GetServerTableMetadataResult {
+			/**
+			 * The generated create scripts for all tables in every database for a single server.
+			 */
+			scripts: string[];
+		}
+
+		export interface ServerMetadataProvider extends DataProvider {
```

Comment:
It would probably make more sense to try and make this specific to copilot - or do you see this being used for other stuff? The term "metadata" is just so overloaded in ADS already, and I would not really consider table scripts to be "metadata" so trying to make it super clear what this is intended for seems like a good idea.